plugin/utils/gapic_utils.py

```diff
@@ -57,40 +79,54 @@ def read_from_gapic_yaml(yaml_file):
     return GapicConfig(collections, fixed_collections, oneofs)
 
 
-def load_collection_configs(config_list, existing_configs):
-    for config in config_list:
-        entity_name = config['entity_name']
-        name_pattern = config['name_pattern']
-        java_entity_name = entity_name
-
-        overrides = config.get('language_overrides', [])
-        overrides = [ov for ov in overrides if ov['language'] == 'java']
-        if len(overrides) > 1:
-            raise ValueError('expected only one java override')
-        if len(overrides) == 1:
-            override = overrides[0]
-            if 'common_resource_name' in override:
-                continue
-            java_entity_name = override['entity_name']
+def find_single_and_fixed_entities(all_resource_names):
+    single_entities = []
+    fixed_entities = []
 
-        if entity_name in existing_configs:
-            existing_name_pattern = existing_configs[entity_name].name_pattern
-            if existing_name_pattern != name_pattern:
-                raise ValueError(
-                    'Found collection configs with same entity name '
-                    'but different patterns. Name: ' + entity_name)
+    for collection in all_resource_names:
+        name_pattern = collection['name_pattern']
+        if '{' not in name_pattern:
```

Comment:
What about `projects/*`?

Reply:
oooh good catch
fixed and added unit test for that